scripts/build.sh

```diff
@@ -0,0 +1,4 @@
+# Build Script
+cargo build
```

Comment:
If we are building to release we should do a `cargo build --release` which has all sorts of optimizations and will reduce the binary size. :)